rs/http_endpoints/nns_delegation_manager/src/nns_delegation_reader.rs

```diff
@@ -57,6 +60,29 @@ impl NNSDelegationReader {
             .map(|builder| builder.build_or_original(canister_ranges_filter, &self.logger))
     }
 
+    /// Returns the most recent NNS delegation known to the replica together with some metadata.
+    /// Consecutive calls might return different delegations.
+    /// Note: on the NNS subnet this always returns `None`.
```

Comment:
This part isn't clear to me. If the delegation on the NNS subnet is always `None`, how do we know if the certificate has expired or that the API version is as expected?